crates/ruma-events/src/room/message/audio.rs

```diff
@@ -88,6 +91,21 @@ impl AudioMessageEventContent {
     pub fn info(self, info: impl Into<Option<Box<AudioInfo>>>) -> Self {
         Self { info: info.into(), ..self }
     }
+
+    /// Returns the caption for the audio as defined by MSC2530.
```

Comment:
Let's not mention a MSC but link to the spec instead: https://spec.matrix.org/latest/client-server-api/#media-captions

The same applies to the other methods.